tests/db/dbServiceTest.spec.ts

```diff
@@ -0,0 +1,33 @@
+const dbService = require("../../db/dbService.ts");
+
+describe("DB Tests", () => {
+
+  afterEach(async ()=>{
+    await dbService.cleanUpTestData();
+  });
+
+  it("Can connect to the the db", async () => {
+    expect(await dbService.pingDb()).toBe("DB Auth successful");
+  });
+
+  describe("Users Table", () => {
+    beforeEach(()=>{
```

Comment:
Something missing?

Reply:
After discussing @ursinewalrus gonna leave this here so we can add in this test later